package.json

```diff
@@ -186,6 +186,61 @@
       "type": "object",
       "title": "Azure IoT Edge configuration",
       "properties": {
+        "azure-iot-edge.version.edgeHub": {
+          "type": "string",
+          "default": "1.0",
+          "description": "Set the edgeHub image version, which will be referenced in deployment manifest."
+        },
+        "azure-iot-edge.version.edgeAgent": {
+          "type": "string",
+          "default": "1.0",
+          "description": "Set the edgeAgent image version, which will be referenced in deployment manifest."
+        },
+        "azure-iot-edge.version.tempSensor": {
+          "type": "string",
+          "default": "1.0",
+          "description": "Set the tempSensor image version, which will be referenced in deployment manifest."
+        },
+        "azure-iot-edge.version.cmodule": {
+          "type": "string",
+          "default": null,
+          "description": "Set the template version of C module template, which will be refernced during scaffolding C module. null for latest."
```

Comment:
`Set the template version of C module template` -> `Set the C module template version` for simplicity and alignment with image version. Same for other templates.